docs/migrate/migration-overview.md

```diff
@@ -90,9 +90,9 @@ If for some reason you can't use the Data Migration Tool but still want a higher
 
 - Version control information and history (original dates and users added to comments)
 - Work items (standard and custom) and history (retains the original dates and users)
-- Test Plan, Test Suite, Test Cases and Test Results
-- Iteration, Area Path, Group, Team, User
-- Dashboard, Query, Widget, and Pipeline
+- Test plan, Test suite, Test cases and Test results
```

Comment:
Thanks for following up on this! Please uncap the three "Test"s, so just the first is capitalized. Same goes for the other updates. Thanks!